syntax = "proto3";
package Protobuf;
option csharp_namespace = "Communication.Proto";

message ChatMessage { string Message = 1; }
